solutions/devsprint-pedro-menezes-1/DeliveryAppChallenge/Modules/Product/Scenes/LeonardoAlmeida/Authentication/Login/LALoginViewController.swift

```diff
@@ -0,0 +1,240 @@
+import UIKit
+
+class LALoginViewController: UIViewController {
+    
+    @IBOutlet weak var heightLabelError: NSLayoutConstraint!
+    
+    @IBOutlet weak var errorLabel: UILabel!
+    
+    @IBOutlet weak var emailTextField: UITextField!
+    @IBOutlet weak var passwordTextField: UITextField!
+    
+    @IBOutlet weak var loginButton: UIButton!
+    @IBOutlet weak var createAccountButton: UIButton!
+    @IBOutlet weak var showPasswordButton: UIButton!
+    
+    var passwordFieldIsSecure = true
+    var formLoginIsInvalid = false
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+       
+        let userIsLogged = verifyIfUserIsLogged()
+        if(userIsLogged) {
```

Comment:
```suggestion
        if userIsLogged {
```
Só uma sugestão, não precisa dos parenteses aqui :D